docs/generated/checks.md

```diff
@@ -2,6 +2,22 @@
 
 KubeLinter includes the following built-in checks:
 
+## check-untrusted-images
```

Comment:
This doesn't seem like a great built-in check. How about just having a check for `latest` tag alone? That seems more focused and useful, and we can enable it by default too.

Reply:
Re: `How about just having a check for latest tag alone?`

Would it be useful to break this check into 2 different checks then?

- Keep the first to just check for `latest` tag alone & then add it as a default built-in check.
- And then make a second one that checks for the container image source registry?

Also, do you have pointers or any suggestion to improve the image source registry check part?

Thank you!

Reply:
@viswajithiii,, I've implemented the current PR for the `latest` tag check alone. I've added this check as a `built-in default` check, as you suggested above.

Also, have added a `template_test.go` file with required unit tests.

---

Re-iterating: `do you have pointers or any suggestion to improve the image source registry check part?`

Thank you!